frontend_build/src/read_bundle_plugins.js

```diff
@@ -24,8 +24,6 @@ var libs = function(kolibri_name) {
     'logging': kolibri_name + '.lib.logging',
     'vue': kolibri_name + '.lib.vue',
     'kolibri': kolibri_name,
-    'core-base': kolibri_name + '.lib.coreBase',
```

Comment:
What is the purpose of these changes? This will result in the core-base and content-renderer code being seperately bundled into every app that uses it, rather than being in the core-app.